app/components/collapsed_section/style.scss

```diff
@@ -1,11 +1,16 @@
 @import "../base.scss";
 
+$govuk-border-width-narrow: 5px;
```

Comment:
Is this a real govuk class? if so, why are we redefining it? If not, we shouldn't be using the `govuk` namespace.

I'd probably look inside the design system sass to see what variables they use.

Reply:
this is from the design system, it's configured with a default value but can be overridden on a per project basis. The default value does not match the same width on the prototype (4px vs 5px).